src/app/IdeaForm.js

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import styled from 'styled-components';
+import uid from 'uid';
+
+const StyledLabel = styled.label`
+  display: grid;
+  margin: 0 0 5px 0;
```

Comment:
You can use ``margin-bottom: 5px`` here.

Reply:
Yes I will